doc/distribution.md

```diff
@@ -0,0 +1,104 @@
+# Content Distribution
+
+When an editor publishes content in Writer, it gets distributed across the web. In this doc, we break down the distribution of this content.
+
+![distribute-content-meme](http://files.artsy.net/images/screen-shot-2017-05-25-at-40907-pm.png)
+
+## [Artsy.net](https://www.artsy.net/articles)
+
+We publish all of our content on [Artsy.net](https://www.artsy.net/articles). Each piece of content belongs to a Channel which helps us decide how an article should look.
+
+Because we have multiple channels, each piece of content on the site gets treated differently depending on which channel it belongs to. For example, we have [promoted content](https://www.artsy.net/article/artsy-school-s-out-for-the-summer-paul-winstanley-s-spare-portraits-of-empty-art-schools) written by Artsy staff, [partner-written articles](https://www.artsy.net/waterhouse-and-dodd/article/waterhouse-dodd-interview-juliette-losq), and [Life at Artsy](https://www.artsy.net/article/daniel-doubrovkine-what-do-software-engineers-do) posts which you can see all have different visual treatments.
+
+In [Force](https://github.com/artsy/force), we've separated the main article body rendering to a [component](https://github.com/artsy/force/tree/master/desktop/components/article). This helps us embed articles anywhere on the site, like on a [fair article](https://www.artsy.net/the-armory-show-2017/info/about-the-fair) or artist bio.
+
+>##### Aside on Channels
+> We typically think of Channels as groups of people. A [Channel](https://github.com/artsy/positron/blob/master/api/apps/channels/model.coffee) contains some metadata and an array of Users. Users can be in multiple Channels. Depending on which Channel you're writing articles in, you'll see different layout options and administrative settings. A long time ago, we only had Users and our entire Artsy Editorial staff had to use a single login.
+
+Bored? Here are some great Artsy Editorial (Channel) articles:
+- [Are Video Games Art?](https://www.artsy.net/article/artsy-editorial-are-video-games-art)
+- [What Happens If You Break an Artwork?](https://www.artsy.net/article/artsy-editorial-break-artwork)
+- [GIPHY Is Helping Get Artists’ Works Viewed 100 Million Times](https://www.artsy.net/article/artsy-editorial-giphy-artists-works-viewed-100-million-times)
+- [What Makes "Bad" Art Good?](https://www.artsy.net/article/artsy-editorial-bad-art-good)
+
+Articles are responsive, so we look good everywhere:
+
+![responsive-article](http://files.artsy.net/images/screen-shot-2017-05-25-at-13624-pm.png)
+Image from [ami.responsivedesign.is](http://ami.responsivedesign.is/)
+
+Speaking of Facebook...
```

Comment:
Ha yea it's in the headline of the image. Try reading from [here](https://jbt.github.io/markdown-editor/#zVpdd9zGkX3nr+iVH6RRCAwpmVask2MvJcsSs1bCmNrVyXL3aHqAxqBNAA2hAQ7HT/s3krf8tfyS3FvdAIaSGMfej7M+R6YINKqr6+PWrWp9pp67pjdNr76xvu/seuitaw4O3pamUbpRJre961Q7rCvrS+NVFpfbRr3tbG+6Q2V7tTG9V/koweRKZ53zXvWlUVuzTtVZg79brHHZIZ6odWf0FX7bNrIm39tcuSKsjVulBwf/dDnLTuLjpDa1+c8HZd+3T5fLwlbGp7rr/S5tTL+0td4Yv/RZZ0yT+NL1yaOj4yfJ0Uny6CTRffL50ZdHT5K2Tttmszg4+OwzdXk6fh2keojdbrd7QvE3m2EfrH9rRpsoXVVU2Q3dZBwc4mdIS9ULnZWqtSYzlDRKWZvKNRsY0Smtnpe6aUyltqXF2tJUrVcD7GkymxtVui29FUUqnHeoclU5dwXrPTOZHjwdoUp9bVQ9VL1tsSwLMv2hMp9UwAXneLg5eLiH1+jd3BaF6bCk2kGD1jS5bTZcHrSLchkY8xlS9S0CydzoGlsfTspctp2rHYXGXX/CXEt5kvisdK5KfOKGPilcl0DRxA91bbqk1UOVbG3je91UBmsT3+rOJK3r+k7bHh8VianbfpdAWBTlF2qLeIYGar1T4jwFAUVxCBWxrIHgccHoubtU3cJIHVzgTaKbPMldnk/q773j88Ti0N21Ndvkh6GyBvKTyvn3i0P4M1eX39nCKN0HhX7KNLlurKkgd1h37vrKNibZloj13CXeFf2WRjDNBs9N5/F0oVrn4dbgtZ0bVIYg8sZITIt7Jk+ra+sHXYUQqPHAI7KQ1ZfwamZmzTa2L4d1mrk6eGpZ8P2CDr8Ped7AmBJDjKxa2zlo1y7fKWyVm47RJFF/CTmta26Fxad3WEIxs6y1hzmXufFXvWuX09d+tBGS7Q2xZUogU68JV9GjMPpuW+LA+7F/qCp7JU+gUKFtNy6/yx+MRd3VrtsRebaCPEvbFG6p14xXvqechXJBlkeiWAeDfvUZ/1OnnkmNDWPW+4OvFBCn37U2g2t2xMfmiqk6LlDaq03nBhwKT1vjkGSpOlWXccHfMR9iwPada5bryq1HC+rW4k/rlyNGLGuXmwqfIR7MQpIVzvPKuxp+NL3Oda8lZgWHOr2jIv/qEWlp+CHBtTasHBMCjeqn6puPYWSEPATmfXiE2cf3k69scyjvEKuM2TlUK43HgK+W5cQHpfLaNqwhurcShj1leZqIAKV6i2PojRNgcg1MXOo86s3vie4QbaFHwIYXUhgtMkJQQpYjZImzGlHTbHC6yiHXCMCuM/nX6hXjCjkYbLZhIn0k7EE89GI65tODhMXEqH9DTDj1Utc4Oj77+h8DSjOKTpj915SRbCiD2Leg7LeACPUKzjaw1Vmh/ggceCblGf7CRlvXXf3szaS+cwt+Ldu8PDt/9Ud15mGGqqUfXxo5Ppzi//pff1JvsdDjkGaLfDw+OlKvbVWRDryBa+6E2rv239i2FHin+IRK+ORaZCeQndRBdkK3+9kKr/UVjHvvmc7vUTX10rn85x9d51JXNvgYTOF0ghb4sDMegOQRgoeIAsYaa7TiUmWuTRfA5ykZz7w0uQU3v4TsHD/+4tHnE9k543JVoO6qS13bdN4JuGk3TWr9tNUd75c42EULF0vCFupbnZk12UaaCpcafwfvYxUOjhYrPPj22dnpggcs1lbPtu2z3mRl1g1NVkL54zQcEp7LWyjgBbDw4mR5fLQMtb0fDePh7iLFn9tnezMJ3NtmejYK/GJ59Ggp+Hxb5jKwPCY0C8HlRyc5PT+b5ebwXuVaol0x2oIbgO/6j0UDXRdECw/IU43Z7tefXA0tsNQI75vsqn7fhACZqHg+sz19HQjVlc1YFAogUGfeDwZlZU/Gfa9WUZN3434rgFreOpAQtUVdYPnDpxlBlMCPqEoFDwjdWWkoHsAqrB4Ajy0LIuJHDP6SUfyLS85kJik5y5n6T9XHhuK8In0Zjwd3rFiISpeLFfeUXA+2yifN9zqYy9CT4AwZknf3j2i8TzM+1niPcHxo6kXIjBfgPID5C/y/L7sBQRaelCjg5tpVSDJoJuSHwipzo/wOu9VyqHJAdKOIFYXNVN4hJTsF+jMXkEMcNhzT8mMSNR2aqubW4W+5S9gGDIFE63vQXvk0VRcWMKk7iSEkrYRYO6DjGYXEEOGC8TyHUilfXJzDRt2V6QOTc1WqYi6NCxGNl6soaLVneLQZPUxm8tTHlSGPpvwSq8cPl4s5fqFE7qRxlL16QDmq/jVcQEQARiGEAyNAmS3Ak9FxgEvB3Ktr3SETelaAim0OaCEjS7IhG3wPMJEPPC2w1XNnFNWIBHFV9nW1IqdbAYnhkXcVqPbqDnUk9DTwnw3u7daImRiEic2ZfK32BAf16s3r75Jcwe9+apXNhASaEulY0iSynqwfhC9uDPoXgIqsNhJwa+1x9g/CYtRLjjv5TB61HcpMZyEMiUMnb425wm+AL6/APJt8aINoUmwzsiVh9VHNsDFk3dqbR+KKkIw/Qa3vKrqtbnSt8YMhkpgb+E/qJkt/lUsbRtb9ZeLwo0vWdoPA7RMqn3jKQBHh/glPNTYJ7BPQWKONZZX2TxXLlnwiJ/nl9fiLo0ePT+bhw1cq0P5Xlqrs4MZI/2UbvD5F4jc5YkrtjCYjjUQV+ZdvO3TPsTZE3hm9cOo9o3krswAQb8aQlA06K4QsVEN4gfQcIgRagCUiqDOBKLEPFrEANPp8D2MkuCxzDiX+HbrZzr+b6hmX1wzX4G48ZrQMoRP4Pf4KvVxmGY3fROSl+19PePHg+fev0TBeoFFU56ZjSvSmQo9Tm6fq3jeD2QccgqXZdAJyh2oKCISry4dM4I3K/uFUEHZD5YQdfP4rJXqPk6rH2EPX6J6ajAyeYcBtjpMThc5hYD2GnvAzotf0eN2oR7HPUk+kffDxs/QegR7u/MNgBgMCWKBW0FuCqxapFYoTNMwojv3aADqKk8+liQUg+h9MEaU3H26tyEDhkXo3vSSptOqg2mAATS4S545CZCB2c83cpS1C4t4JS+YDzNMbdunSFD9YiZR3InMlwlZBWnwEt0FzxJpeV+Im4OnsqhA1KLhsFVHIIfyqQYKFdk/GT35//hTHWGxNUIZi0eehxwjHCba2Daj2ntaOeZCj7gPMj49PX6MWvZEJorz/b6Ts46OTR3PKspSjO2C2nb4+J1PkdGPNsOh1diW+uvStYTX3fjB7jFr2jATDOlKgTeChx8ekogiSzl3je4DZxiQiI2FJSBDlbfm+QtFjydc0WyUwcR4oIb4JUyOJYcIDzAbzceaA/mvS9gNsrVts+APyMHXdBsJHL7FZtpuyx2ZbyrlNKAR/tnFSsnZYkAIen8HueTkNcOgXbKjOwwaqGyaOS6hSq9+wyqm//vkvX63GcsBTi3BQAfUfvzkGqCAeOMxkhy15Jgkcz3OBM4LMs+T48PE0aCgE59ZUSQYLK3Qz7WrUS6Cq2JuafqJRlwkQM+3DNn1vBsED/m/26FT6/6hPn7b6/9ur37bG/3i/HsQfvCnHCKhDR6Q9B0mX23KvR6ghYqiFnf5z1uk6jCG3PErdJqASBbqE5En+qCiOi89/ffLlk4UkA+kV3sxM/eToX56p5xcXKMk1iqo3pvYRRvm6RG7FVi43iD1igUBfFqfxmrcNr88JFZllY+D7HWKzNOanp6b7Ddg4Nd1vaZazLLF9yt8XgSuTFKLj3EWECSi+t54asw9FkY8XQOSy3nDgTPuOk7bGmMAagGosYjRFQApJdJQKG28MOsMhoCH1YHtltKRw2EMgZ+ad/CZ0XN9fXKAIz5c0FZvrfioycz8dC0ylaWzogSryAZZE5nv5O3JeyJUidle8dd4vSRb32ORIwwWVZHBOkVp43A7u73sg2osbGXigfAzrBL74RZNv8SE0WIRIGbp4WyN9hBDtUX166bQljZFTaTY5FUiHRs12w6bk4J7OQloGVGcx8BHAZTKy9/WnZyKp5grRu7LrTne7qYHLXTZMNWX5UuxTLZ+zgrXsXZYU+w5i331vCvTUublJWTMWYwAEH5+iqdshYP3Bwe9AKCQ8w4UDGl22BEyTj+7pDoXe5zKHEyJi+/t+POnXB4CZhw8vz0G4TZXupgLe8kFa7RYPHz4NkzuWY+RiJRDFeXDv5bYgdmXYHyuAJJkJIsdSPCo9m0yPj9KNrKHNZB9uUaM6KZCAZmBXTjWllHaGF1s+iP4OjNx0ewIlHCt5OgnDzzU6Uz/P4YWReZgmYysQRtPcMu+sMNNDAKG43zYyY8eysN+F2dB9IDLznn56Jrv1Mfgb9BoaO/lS6ELAPy1jLV52Dj64TboUkoRoifA+YoHkzOHetOHlaaQ8bOsEcb4HrSxt0YdJhByMF19y5ZaqYB/GbSOseySjFD5+iZBimzJX+zfoDNjAICdPbdeT2+5ZOD4R60q30210Y380oldb8UYtNJiSit0g10iyKwhW42FPEPzOtn8PrD85MAtf+eVGvxuVeDeKTH/wi0iG2eaM5KrUVSEPQo8eJITGZCfNgJ9nCZMDAt06lfNw1ijX+vU4pZJ9hQeNhojtz8xK0QlZFPWDg4cP35Zg566536szubSZWalMY/w02hNoS+YubqXmUIuR+vXDhwcHe0+lmOiMyLHv1r0tbOhvLuRzJfOXC9NxiNbvWhO600/s7PfW/1Zf62i2t2YtV+P8No7ziTjxJihUZ85j4pXnKIZ59Uk2PubN5PbpshY2XMbsXAZgQCleuutwc8zJIi3Lmspy9ZFlZ1fCuucRvsR6L128Z+W8aw+5kaSp4Bz6xEkdqWpyY7z80fz7jzeVP36/XSe84VybrrGNTTIN1AaHlmFL69Af8K69HAB+iTc3AHTb7xLbcObSmCXaRbn/oEnomHOh7yD8sZu/aUMzEnJ8Wjp0VQSLeA0pHyMlarUixU/nSIO48cARKJUfQFMqVgmitWukaVDNUK+JIJQ/m06MxExxnLgSC+JlSJwihOKwe6ru0c4SgGF8Xsm/HuAcE/u13Jd5AH9ir9DGMPuzK8FW7+Q0YNY+gPw4adDEjwzYVeVx0Dv42PZNY21wIbAvDh14ZtiEMwAWOsmAUNyiW0QCR7lyrLdRBTk/4yh0ZuBybB8ZsQEk9h6ED7w6PoldWfjk8dH4q1j+jBe4LMDkT+KCaQKQuapiIyi6yeS5N3cpA7dO38lIFE62I1aOs9tXQNYfXQN+Oc9uN5rFQAZDHZmebPZgnjnqDbAxcFYtcyn4bIOKVMVpTECJXZixkB1J83dtdRw7/A0=) instead. But actually I'll remove this line because it's not directly mentioned.